tests/system/test_cluster/test_correct_merged_file_generation/test_correct_merged_file_generation.py

```diff
@@ -0,0 +1,197 @@
+'''
+copyright: Copyright (C) 2015-2023, Wazuh Inc.
+           Created by Wazuh, Inc. <info@wazuh.com>.
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+type: system
+brief: Wazuh manager should be able to create merged.mg file in order to share files with group of agents.
+       In order to do it, when new files are present in any directory in /var/ossec/share/,
+       those files must be monitored and to be taken in consideration by merged.mg
+tier: 1, 2
+modules:
+    - enrollment
+components:
+    - manager
+    - agent
+daemons:
+    - wazuh-authd
+    - wazuh-agentd
+os_platform:
+    - linux
+os_version:
+    - Debian Buster
+references:
+    - https://documentation.wazuh.com/current/user-manual/reference/centralized-configuration.html
+'''
+
+import os
+import pytest
+import time
+from wazuh_testing import T_1, T_10
+from wazuh_testing.tools import WAZUH_PATH
+from wazuh_testing.tools.file import read_yaml
+from wazuh_testing.tools.monitoring import HostMonitor
+from wazuh_testing.tools.system import HostManager
+from wazuh_testing.tools.file import replace_regex_in_file
+from system import (assign_agent_to_new_group, clean_cluster_logs, create_new_agent_group, delete_agent_group,
+                    restart_cluster)
+
+# pytestmark = [pytest.mark.cluster, pytest.mark.one_manager_agent_env]
```

Comment:
```suggestion
 pytestmark = [pytest.mark.cluster, pytest.mark.one_manager_agent_env]
```

Reply:
Changes done in https://github.com/wazuh/wazuh-qa/pull/4129/commits/5134d4c56b8ee81f07f1d17aec935dbd80a0d6d2